src/app/containers/Ad/Amp/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const constructAdJsonData = ({ id, service }) => {
+  const data = {
+    targeting: {
+      slot: 'leaderboard',
+      asset_type: 'index',
+      story_id: id,
```

Comment:
- Confirm what all of these parameters are actually for and what they mean

Reply:
- ‘data-slot’ : will contain the ad unit ‘/4817/bbccom.test.site.amp.news’
- ‘width’/‘height’ : for placeholder, example ‘width=“300” height=“250"’
- ‘type’ : will always be ‘doubleclick’
- ‘slot’ : will either be leaderboard, mpu or mpu_bottom
- ‘asset_type’ : story or index
- ‘story_id’ : article id when asset_type is story
- ‘channel’ : WS site, example mundo or arabic
- ‘sectn’ : url section, example ‘noticias’ for https://www.bbc.com/mundo/amp/noticias-internacional-51518722
- ‘subsect’ : url subsection, example ‘internacional’ for https://www.bbc.com/mundo/amp/noticias-internacional-51518722

Reply:
- `data-slot’ : will contain the ad unit ‘/4817/bbccom.test.site.amp.news’
- ‘width’/‘height’ : for placeholder, example ‘width=“300” height=“250"’
- ‘type’ : will always be ‘doubleclick’
- ‘slot’ : will either be leaderboard, mpu or mpu_bottom
- ‘asset_type’ : story or index
- ‘story_id’ : article id when asset_type is story
- ‘channel’ : WS site, example mundo or arabic
- ‘sectn’ : url section, example ‘noticias’ for https://www.bbc.com/mundo/amp/noticias-internacional-51518722
- ‘subsect’ : url subsection, example ‘internacional’ for https://www.bbc.com/mundo/amp/noticias-internacional-51518722